src/app/forms/templates/test-records/section-templates/vehicle/contingency-default-psv-hgv-light-vehicle-section.template.ts

```diff
@@ -40,9 +42,10 @@ export const ContingencyVehicleSectionDefaultPsvHgvLight: FormNode = {
       name: 'euVehicleCategory',
       label: 'EU Vehicle Category',
       value: '',
-      disabled: true,
       type: FormNodeTypes.CONTROL,
-      width: FormNodeWidth.XXS
+      editType: FormNodeEditTypes.SELECT,
+      options: getOptionsFromEnum(EuVehicleCategories),
```

Comment:
we want our templates to stay pure JSON. One way for them to stay pure JSON would be to stick the eu vehicle categories in the reference data endpoint.

Reply:
That would be good, but it's well out of scope for this work. I'll add a CI ticket.

Reply:
This is true, but we haven't got to fully refactoring these, more need done too

Reply:
May be true for tech record templates, however all test records templates are valid JSON.

Reply:
Are you suggesting we get the EU Vehicle Categories moved to the reference service and update VTM accordingly or is there a simpler interim solution?

Reply:
And do you think it's important enough that the PR should not be merged until the change is made?

Reply:
Ticket created to [Move EU Vehicle Category to the reference data service](https://dvsa.atlassian.net/browse/CB2-7231).